src/FunctionsProxy/Adapter/RoundRobin.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace FunctionsProxy\Adapter;
+
+use FunctionsProxy\Adapter;
+
+class RoundRobin extends Adapter
```

Comment:
While we use the adapter pattern, this is not really and adapter, but an algorithm.